src/containers/AuthorsList.js

```diff
@@ -48,25 +44,20 @@ class AuthorsList extends React.Component {
       return newValue
     }
     const authorsArray = _.map(_.values(this.props.entities.authors), iteratee)
-    let isFinish = (this.props.finalPage <= this.props.currentPage) ? true : false
     return (
       <div className={styles['author-list-container']}>
         <ShownAuthors filteredAuthors={authorsArray} />
-        <LoadMore isFinish={isFinish} onIncrement={_onIncrement} />
+        <LoadMore isFinish={this.props.isFinish} onIncrement={_onIncrement} />
       </div>
     )
   }
 }
 
 function mapStateToProps(state) {
-  const currentPage = _.get(state, [ 'authorsList', 'meta', 'page' ])
-  const maxResultsPerPage = _.get(state, [ 'authorsList', 'meta', 'maxResults' ])
-  const totalResults = _.get(state, [ 'authorsList', 'meta', 'total' ])
-  const finalPage = Math.ceil(totalResults/maxResultsPerPage)
+  let isFinish = !shouldLoadMoreAuthors(state)
```

Comment:
You can move `isFinish` field into redux state.
Make this business logic inside reducer.